packages/react/src/components/List/List.story.jsx

```diff
@@ -493,6 +493,7 @@ export const WithCheckboxMultiSelectionAndHierarchy = () => {
     };
 
     const handleSelection = (items, id) => {
+      action('handleSelect');
```

Comment:
```suggestion
      action('handleSelect')();
```
This action isn't firing in the actions panel.